docs/preset-env.md

```diff
@@ -229,38 +229,64 @@ This option is useful for "blacklisting" a transform like `@babel/plugin-transfo
 
 `"usage"` | `"entry"` | `false`, defaults to `false`.
 
-> This option adds direct references to the `core-js` module as bare imports. Thus `core-js` will be resolved relative to the file itself and needs to be accessible. You may need to specify `core-js@2` as a top level dependency in your application if there isn't a `core-js` dependency or there are multiple versions.
+> When using this options, you also need to configure the [`corejs`](#corejs) option.
```

Comment:
Without this option, the user will see a warning that he should set `corejs` option directly and `corejs: 2` will be used by default, so the current version is correct.

Reply:
This is done to match transform-runtime, which requires the corejs option to load polyfills: since the default should always be the last version but it isn't possible without breaking changes, it is better not to have a default (or warn when it is not set).

Reply:
>  so the current version is correct

I'm perfectly fine with letting the user know that stuff is changing and they _should_ set the core-js version... I'm just advocating we use slightly softer language to not give the impression that they _must_ set the version immediately, is all.

(FWIW, I personally disagree with _forcing_ the option)

> This is done to match transform-runtime, which requires the corejs option to load polyfills

Not sure I agree we need to exactly match behavior from `transform-runtime`.

Reply:
> (FWIW, I personally disagree with requiring the option)

Let's discuss this in the other PR. I'm fine both with requiring or not requiring it.

I can change the wording